packages/server/src/chatbot/chatbot.controller.ts

```diff
@@ -378,6 +384,103 @@ export class ChatbotController {
   // }
 
   // resets all chatbot data for the course. Unused
+
+  // Professor-only: send notification email to all students who asked this question
+  // Body must include oldAnswer and newAnswer, and can optionally include question changes for context
+  @Post('question/:courseId/:questionId/notify')
+  @UseGuards(CourseRolesGuard)
+  @Roles(Role.PROFESSOR)
+  async notifyUpdatedAnswer(
+    @Param('courseId', ParseIntPipe) courseId: number,
+    @Param('questionId') questionId: string,
```

Comment:
```suggestion
  @Post('question/:courseId/:vectorStoreId/notify')
  @UseGuards(CourseRolesGuard)
  @Roles(Role.PROFESSOR)
  async notifyUpdatedAnswer(
    @Param('courseId', ParseIntPipe) courseId: number,
    @Param('vectorStoreId') vectorStoreId: string,
```

Since chatbot questions are stored in two different databases (one in helpme DB, one in the chatbot vectorstore DB. And no it wasn't my decision), it's important to use consistent naming.

In this instance, `questionId` is referring to the vectorStoreId so it should probably be updated